src/sentry/uptime/models.py

```diff
@@ -363,6 +375,24 @@ def create_detector_from_project_subscription(project_sub: ProjectUptimeSubscrip
         organization=project_sub.project.organization,
         source_id=str(project_sub.uptime_subscription_id),
     )
+    condition_group = DataConditionGroup.objects.create(
+        organization=project_sub.project.organization,
+    )
+    # TODO(epurkhiser): Should we be creating a new data condition + group for
+    # every uptime detector, or is there an intention to be able to re-use the
+    # groups for multiple detectors?
+    DataCondition.objects.create(
+        comparison=CHECKSTATUS_FAILURE,
+        type=Condition.EQUAL,
+        condition_result=DetectorPriorityLevel.HIGH,
+        condition_group=condition_group,
+    )
+    DataCondition.objects.create(
+        comparison=CHECKSTATUS_SUCCESS,
+        type=Condition.EQUAL,
+        condition_result=DetectorPriorityLevel.OK,
+        condition_group=condition_group,
+    )
```

Comment:
I think it makes sense to have this as well right?

Reply:
Yeah, we'll want a resolution condition to tell the detector it's okay.

🤔 is this something you think the platform should handle / automate based on the other conditions or do you like having the separate control for setting the `comparison` info?

Reply:
I personally like having it explicit like this. But @wedamija pointed out that actually is the default if no condition matches (OK)

Reply:
Yeah I'm not sure what we prefer. I'm ok with requiring us to be explicit, if that's what we prefer.